pypeit/specobj.py

```diff
@@ -467,29 +467,6 @@ def apply_flux_calib(self, wave_sens, sensfunc, exptime, telluric=None, extinct_
             self[attr+'_FLAM_SIG'] = flam_sig
             self[attr+'_FLAM_IVAR'] = flam_var
 
-
-    def apply_helio(self, vel_corr, refframe):
-        """
-        Apply a heliocentric correction
-
-        Wavelength arrays are modified in place
-
-        Args:
-            vel_corr (float):
-            refframe (str):
-
-        """
-        # Apply
-        for attr in ['BOX', 'OPT']:
-            if self[attr+'_WAVE'] is not None:
-                msgs.info('Applying {0} correction to '.format(refframe)
-                          + '{0} extraction for object:'.format(attr)
-                          + msgs.newline() + "{0}".format(str(self.NAME)))
-                self[attr+'_WAVE'] *= vel_corr
-                # Record
-                self['VEL_TYPE'] = refframe
```

Comment:
Confirm these are being set, i.e. propagated to the header.

Reply:
Same with `FLEX_SHIFT` which was set in the `flexure_interp()` method which may not be called any longer.